<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenGL Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="3">
     <widget class="QtGLViewer" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="8" column="2">
       <widget class="QLCDNumber" name="lcdNumber_2">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>30.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLCDNumber" name="lcdNumber_1">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="enclose_Button">
        <property name="text">
         <string>Enclosing </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="fillRect_chkBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Fill Polygons</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Dilation Size:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="planeToCloud_Button">
        <property name="text">
         <string>Planes to Cloud</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Cloud Density:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSlider" name="dilationS_slider">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSlider" name="thresholdA_slider">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="sliderPosition">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="outerBox_chkBox">
        <property name="text">
         <string>Outer Box</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="analytic_chkBox">
        <property name="text">
         <string>Analytic Intersec.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="4">
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Enclosing:</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>221</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="boundRect_radBut">
            <property name="text">
             <string>Bound Rect.</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="concavH_radBut">
            <property name="text">
             <string>Concav Hull</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="rotdRect_radBut">
            <property name="text">
             <string>Rotated Rect.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="contour_radBut">
            <property name="text">
             <string>Contourn</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>609</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File GLView</string>
    </property>
    <addaction name="loadCloud_Action"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="loadCloud_Action">
   <property name="text">
    <string>Load Plane Cloud</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtGLViewer</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">qtglviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dilationS_slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_1</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thresholdA_slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
